src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js

```diff
@@ -0,0 +1,31 @@
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+import STATISTIC_CATEGORY from 'Interface/Others/STATISTIC_CATEGORY';
+import SpellIcon from 'common/SpellIcon';
+import React from 'react';
+
+class Apotheosis extends Analyzer {
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.APOTHEOSIS_TALENT.id);
+    this.active = false;
+  }
+
+  statistic() {
+    return (
+
+      <StatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.APOTHEOSIS_TALENT.id} />}
+        value={"Value"}
+        label="Apotheosis"
+        tooltip={``}
+      />
```

Comment:
This tooltip is empty.

Reply:
The module is disabled. With as big as the PR is, I want to get in what I can and add the last 2 talents later on. Apotheosis is going to be a pita.

Reply:
Preferably just remove it completely for now then imo 😃